.rubocop.yml

```diff
@@ -17,10 +18,18 @@ Metrics/MethodLength:
 Metrics/BlockLength:
   Exclude:
     - 'spec/**/*.rb'
+    - 'api_auth.gemspec'
 
 Naming/FileName:
   Exclude:
     - 'lib/api-auth.rb'
 
 Style/FrozenStringLiteralComment:
   Enabled: false
+
+Style/StringLiterals:
+  Exclude:
+    - 'gemfiles/*.gemfile'
```

Comment:
FYI: This files is auto-generates and contains double quotes, so need to ignore this warnings for these files.

Reply:
:+1: